packages/app-store/exchange2013calendar/_metadata.ts

```diff
@@ -0,0 +1,26 @@
+import type { App } from "@calcom/types/App";
+
+import _package from "./package.json";
+
+export const metadata = {
+  name: "Microsoft Exchange 2013 Calendar",
+  description: _package.description,
+  installed: true,
+  type: "exchange2013_calendar",
+  title: "Microsoft Exchange 2013 Calendar",
+  imageSrc: "/api/app-store/exchange2013calendar/icon.svg",
+  variant: "calendar",
+  category: "calendar",
+  label: "Exchange Calendar",
+  logo: "/api/app-store/exchange2013calendar/icon.svg",
+  publisher: "Cal.com",
+  rating: 5,
+  reviews: 69,
+  slug: "exchange2013-calendar",
+  trending: false,
+  url: "https://cal.com/",
+  verified: true,
```

Comment:
Be aware of @alannnc changed introducing a new flag to allow multiple installations for an app, which applies to these new Exchange apps: #3045